modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,67 +1,59 @@
+* You must power off the VMware virtual machine before importing it.
 
 .Procedure
 
-. Click *Workloads* -> *Virtual Machines* from the side menu.
-. Click *Create Virtual Machine* and select *Create with Wizard*.
-. Perform the following procedure in the *Basic Settings* screen:
-.. Click *Provision Source* and select *Import*.
-.. Click *Provider* and select *VMware*.
-.. Click *vCenter Instance* and select *Connect to New Instance* or a saved vCenter instance.
-+
-* If you select *Connect to New Instance*, provide values in the following fields and click *Check and Save*:
+. In the {CNVProductName} web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create* and select *Import with Wizard*.
+. In the *General* screen, perform the following steps:
+.. Select *VMware* from the *Provider* list.
+.. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
 
-** *vCenter Hostname*
-** *vCenter User Name*
-** *vCenter Password*
-+
-The wizard connects to the vCenter instance and saves the credentials.
+.. Select a virtual machine or a template to import.
 +
-If you clear the *Save as New vCenter Instance* check box and click *Check*, the wizard connects to the vCenter instance without saving the credentials.
+The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
 
-* If you select a saved vCenter instance, the wizard connects to it by using the saved credentials.
+.. Select an existing flavor or *Custom* from the *Flavor* list.
+.. Select a *Workload Profile* and then click *Next*.
 
-.. Click *VM to import* and select a virtual machine or template.
-+
-The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
+. In the *Networking* screen, perform the following steps:
 
-.. Specify the *Memory (GB)* and number of *CPUs* if you use the default flavor, *Custom*.
+.. Click the Options menu {kebab} of a network interface and select *Edit*.
+.. Enter a valid network interface name.
 +
-Optionally, you can select an existing *Flavor*.
+The name can contain lower-case letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain upper-case letters, spaces, periods (`.`), or special characters.
 
-.. Select a *Workload Profile*.
-.. Click *Next*.
-. Perform the following procedure in the *Networking* screen:
-.. Click *NETWORK CONFIGURATION* and select *Pod Networking*.
-.. Select a *BINDING METHOD*.
-.. Click the &#10003; button to save the update.
-.. Click *Next*.
-. Perform the following procedure in the *Storage* screen:
-.. Click each disk and select a *STORAGE CLASS*. If you do not select a storage class, {CNVProductName} uses the default storage class to create the virtual machine.
+.. Select a network definition from the *Network* list.
```

Comment:
After 'Network' there are also Type (with "Bridge" content, for example), and then 'MAC Address', that are missing here.

Reply:
Is filling in the MAC address a requirement? In 2.1, it was optional

Reply:
OK, added it.